src/resolvers/hosts/hosts.integration.ts

```diff
@@ -227,6 +230,29 @@ describe('hosts query', function () {
             expect(data).toMatchSnapshot();
         });
 
+        test('group_name ASC', async () => {
+            const { data, status } = await runQuery(BASIC_QUERY, {
+                order_by: 'group_name',
+                order_how: 'ASC'
+            });
+
+            expect(status).toEqual(200);
+            expect(data).toMatchSnapshot();
+            // the first host is not expected to have any groups, i.e. null
+            data.hosts.data[0].groups.data.length === 0
```

Comment:
This statement isn't an assertion; it's just a boolean. Based on the assertions around it, it should probably be something like `expect(data.hosts.data[0].groups.data.length).toEqual(0)` or `data.hosts.data[0].groups.data[0].length.should.equal(0)`

Reply:
Thanks and fixed!